actors/common/src/main/java/org/sunbird/metrics/actors/BaseMetricsActor.java

```diff
@@ -394,6 +390,14 @@ public static String makePostRequest(String url, String body) throws IOException
     return result.toString();
   }
 
+  public static String makePostRequest(String apiURL, String body) throws IOException {
+    String baseSearchUrl = System.getenv(JsonKey.EKSTEP_BASE_URL);
```

Comment:
ProjectUtil has method to get config value directly from either env or properties file.